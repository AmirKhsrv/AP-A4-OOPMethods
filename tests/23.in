best_book
best_reviewer
show_sorted_shelf 128 read Sequential Art
credit 130
show_sorted_shelf 39 read Fantasy
best_reviewer
credit 19
show_author_info 722
best_book
credit 63
show_sorted_shelf 51 read Fiction
best_book
show_sorted_shelf 34 currently_reading Sequential Art
best_book
recommend_first_approach 116
show_sorted_shelf 52 read Sequential Art
show_author_info 329
best_reviewer
show_sorted_shelf 58 currently_reading Romance
show_sorted_shelf 10 read Fiction
best_book
show_author_info 1741
show_author_info 625
