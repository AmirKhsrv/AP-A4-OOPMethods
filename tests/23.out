id: 405
Title: Ellen Foster
Genre: Fiction
Author: Kaye Gibbons
Average Rating: 3.42
id: 31
Name: Athena Souri
Place of Birth: Switzerland
Member Since: December 24 2010
Favorite Genres: Romance
Favorite Authors: Carl Sagan, Colleen Clayton, Greer Noble
Number of Books in Read Shelf: 8
Number of Books in Want to Read Shelf: 15
Number of Books in Currently Reading Shelf: 6
Number of Likes: 2455
id: 225
Title: Bet Me
Genre: Romance
Author: Jennifer Crusie
***
id: 7
Title: Dracula
Genre: Classics
Author: Bram Stoker
***
id: 178
Title: How Late it Was How Late
Genre: Fiction
Author: James Kelman
***
id: 190
Title: How to Win Friends and Influence People
Genre: Nonfiction
Author: Dale Carnegie
***
id: 31
Title: Megan Meade's Guide to the McGowan Boys
Genre: Young Adult
Author: Kate Brian
***
id: 376
Title: Season of Migration to the North
Genre: Fiction
Author: Tayeb Salih
***
id: 159
Title: The Goldfinch
Genre: Fiction
Author: Donna Tartt
***
id: 276
Title: The Mad Wolf's Daughter
Genre: Historical
Author: Diane Magras
***
id: 306
Title: The Madness Underneath
Genre: Young Adult
Author: Maureen Johnson
***
id: 256
Title: Unhinged
Genre: Fantasy
Author: A.G. Howard
***
id: 80
Title: We Were the Mulvaneys
Genre: Fiction
Author: Joyce Carol Oates
***
0.014122
id: 26
Title: Kings Rising
Genre: Fantasy
Author: C.S. Pacat
***
id: 343
Title: Knife Sworn
Genre: Fantasy
Author: Mazarkis Williams
***
id: 121
Title: Escaping Reality
Genre: Romance
Author: Lisa Renee Jones
***
id: 128
Title: Grundish and Askew
Genre: Fiction
Author: Lance Carbuncle
***
id: 325
Title: Ilustrado
Genre: Fiction
Author: Miguel Syjuco
***
id: 238
Title: Lamb: The Gospel According to Biff Christ's Childhood Pal
Genre: Humor
Author: Christopher Moore
***
id: 140
Title: Naomi and Ely's No Kiss List
Genre: Young Adult
Author: Rachel Cohn
***
id: 201
Title: Shadow Hills
Genre: Young Adult
Author: Anastasia Hopcus
***
id: 168
Title: The Book Thief
Genre: Historical
Author: Markus Zusak
***
id: 19
Title: The Darkest Kiss
Genre: Romance
Author: Gena Showalter
***
id: 259
Title: The Erotic Dark
Genre: Erotica
Author: Nina Lane
***
id: 315
Title: The Secret History
Genre: Fiction
Author: Donna Tartt
***
id: 397
Title: The Snowman
Genre: Childrens
Author: Raymond Briggs
***
id: 136
Title: Washington's Crossing
Genre: History
Author: David Hackett Fischer
***
id: 31
Name: Athena Souri
Place of Birth: Switzerland
Member Since: December 24 2010
Favorite Genres: Romance
Favorite Authors: Carl Sagan, Colleen Clayton, Greer Noble
Number of Books in Read Shelf: 8
Number of Books in Want to Read Shelf: 15
Number of Books in Currently Reading Shelf: 6
Number of Likes: 2455
0.008446
id: 722
Name: John Shelton Jones
Year of Birth: 1967
Place of Birth: Chile
Member Since: October 8 2013
Genres: Fiction
Books:
id: 405
Title: Ellen Foster
Genre: Fiction
Author: Kaye Gibbons
Average Rating: 3.42
0.009170
id: 141
Title: Cassandra at the Wedding
Genre: Fiction
Author: Dorothy Baker
***
id: 372
Title: Terra Feliz: Vanity
Genre: Fiction
Author: Ginny Bryce
***
id: 233
Title: Beauty's Release
Genre: Adult Fiction
Author: A.N. Roquelaure
***
id: 173
Title: Dangerous Attraction Romantic Suspense Boxed Set
Genre: Romance
Author: Kaylea Cross
***
id: 174
Title: Five Days at Memorial: Life and Death in a Storm-Ravaged Hospital
Genre: Nonfiction
Author: Sheri  Fink
***
id: 404
Title: Monster Hunter Legion
Genre: Fantasy
Author: Larry Correia
***
id: 270
Title: Siddhartha
Genre: Classics
Author: Hermann Hesse
***
id: 236
Title: Slow Burn
Genre: Romance
Author: Sam B. Morgan
***
id: 303
Title: The Good Luck Charm
Genre: Romance
Author: Helena Hunting
***
id: 300
Title: The Gray Wolf Throne
Genre: Fantasy
Author: Cinda Williams Chima
***
id: 68
Title: Wreck This Journal
Genre: Nonfiction
Author: Keri Smith
***
id: 166
Title: You Can Heal Your Life
Genre: Self Help
Author: Louise L. Hay
***
id: 405
Title: Ellen Foster
Genre: Fiction
Author: Kaye Gibbons
Average Rating: 3.42
id: 126
Title: Diva
Genre: Historical
Author: Jillian Larkin
***
id: 187
Title: Endure
Genre: Young Adult
Author: Carrie Jones
***
id: 402
Title: Fairy Tale
Genre: Young Adult
Author: Cyn Balog
***
id: 392
Title: From the Mixed-Up Files of Mrs. Basil E. Frankweiler
Genre: Fiction
Author: E.L. Konigsburg
***
id: 100
Title: Last First Kiss
Genre: Romance
Author: Diane  Adams
***
id: 153
Title: Release Me
Genre: Romance
Author: J. Kenner
***
id: 380
Title: Starlet's Web
Genre: Young Adult
Author: Carla J. Hanna
***
id: 315
Title: The Secret History
Genre: Fiction
Author: Donna Tartt
***
id: 55
Title: Twelve Sharp
Genre: Mystery
Author: Janet Evanovich
***
id: 405
Title: Ellen Foster
Genre: Fiction
Author: Kaye Gibbons
Average Rating: 3.42
id: 1
Title: Forging Zero
Genre: Science Fiction
Author: Sara  King
Reviews:
id: 416 Rating: 2 Likes: 1 Date: April 4 2014
id: 708 Rating: 4 Likes: 0 Date: October 19 2010
id: 866 Rating: 0 Likes: 0 Date: July 5 2016
id: 889 Rating: 1 Likes: 0 Date: December 21 2017
id: 1572 Rating: 2 Likes: 0 Date: April 18 2015
id: 1766 Rating: 2 Likes: 0 Date: June 8 2013
id: 2152 Rating: 1 Likes: 3 Date: January 2 2019
id: 2948 Rating: 1 Likes: 0 Date: October 14 2017
id: 3017 Rating: 4 Likes: 0 Date: July 25 2018
id: 4267 Rating: 0 Likes: 1 Date: January 14 2015
id: 4640 Rating: 1 Likes: 0 Date: September 9 2016
id: 5212 Rating: 4 Likes: 0 Date: January 12 2010
id: 5554 Rating: 5 Likes: 0 Date: January 27 2015
id: 6186 Rating: 0 Likes: 0 Date: July 14 2018
id: 6356 Rating: 5 Likes: 3 Date: December 13 2015
id: 6574 Rating: 4 Likes: 9 Date: March 15 2018
id: 7151 Rating: 3 Likes: 5 Date: July 18 2014
id: 7884 Rating: 3 Likes: 0 Date: August 29 2013
id: 9223 Rating: 3 Likes: 0 Date: October 20 2011
id: 9810 Rating: 2 Likes: 0 Date: June 5 2016
id: 11056 Rating: 2 Likes: 0 Date: October 16 2012
id: 11581 Rating: 4 Likes: 4 Date: July 24 2015
id: 11997 Rating: 5 Likes: 3 Date: November 19 2015
id: 12488 Rating: 4 Likes: 0 Date: September 17 2019
id: 12559 Rating: 5 Likes: 0 Date: March 5 2010
id: 13030 Rating: 2 Likes: 2 Date: November 23 2017
id: 13083 Rating: 2 Likes: 0 Date: August 8 2019
id: 13266 Rating: 2 Likes: 0 Date: January 7 2015
id: 13897 Rating: 1 Likes: 0 Date: February 28 2014
id: 15083 Rating: 5 Likes: 0 Date: July 24 2018
id: 16304 Rating: 0 Likes: 0 Date: August 10 2018
id: 111
Title: Above World
Genre: Fantasy
Author: Jenn Reese
***
id: 179
Title: Ghostgirl
Genre: Fantasy
Author: Tonya Hurley
***
id: 3
Title: How to Win Friends and Influence People
Genre: Nonfiction
Author: Dale Carnegie
***
id: 293
Title: It's Kind of a Funny Story
Genre: Young Adult
Author: Ned Vizzini
***
id: 123
Title: Lead
Genre: Romance
Author: Kylie Scott
***
id: 108
Title: Other
Genre: Young Adult
Author: Karen Kincy
***
id: 165
Title: Shadow Scale
Genre: Fantasy
Author: Rachel Hartman
***
id: 43
Title: Soulless
Genre: Romance
Author: T.M. Frazier
***
id: 54
Title: The 19th Wife
Genre: Historical
Author: David Ebershoff
***
id: 138
Title: The Heart Is a Lonely Hunter
Genre: Fiction
Author: Carson McCullers
***
id: 329
Name: May Sarton
Year of Birth: 1993
Place of Birth: Guinea
Member Since: November 27 2015
Genres: Poetry
Books:
id: 31
Name: Athena Souri
Place of Birth: Switzerland
Member Since: December 24 2010
Favorite Genres: Romance
Favorite Authors: Carl Sagan, Colleen Clayton, Greer Noble
Number of Books in Read Shelf: 8
Number of Books in Want to Read Shelf: 15
Number of Books in Currently Reading Shelf: 6
Number of Likes: 2455
id: 299
Title: Switch! The Lost Kingdoms of Karibu
Genre: Fantasy
Author: Karen  Prince
***
id: 11
Title: When Breath Becomes Air
Genre: Nonfiction
Author: Paul Kalanithi
***
id: 128
Title: Grundish and Askew
Genre: Fiction
Author: Lance Carbuncle
***
id: 314
Title: An Inconvenient Truth: The Planetary Emergency of Global Warming and What We Can Do About It
Genre: Nonfiction
Author: Al Gore
***
id: 268
Title: Angels Flight
Genre: Mystery
Author: Michael Connelly
***
id: 32
Title: Blackout
Genre: Horror
Author: Mira Grant
***
id: 267
Title: Damage
Genre: Romance
Author: P.J.  Adams
***
id: 298
Title: Flat-Out Love
Genre: Romance
Author: Jessica Park
***
id: 83
Title: Innocents
Genre: New Adult
Author: Mary  Elizabeth
***
id: 338
Title: Jurassic Park
Genre: Science Fiction
Author: Michael Crichton
***
id: 394
Title: Radio Shangri-La: What I Learned in the Happiest Kingdom on Earth
Genre: Travel
Author: Lisa Napoli
***
id: 19
Title: The Darkest Kiss
Genre: Romance
Author: Gena Showalter
***
id: 255
Title: Then Came You
Genre: Romance
Author: Jill Shalvis
***
id: 286
Title: Three Daves
Genre: Romance
Author: Nicki Elson
***
id: 63
Title: When We Wake
Genre: Young Adult
Author: Karen Healey
***
id: 405
Title: Ellen Foster
Genre: Fiction
Author: Kaye Gibbons
Average Rating: 3.42
id: 1741
Name: Walter Isaacson
Year of Birth: 1967
Place of Birth: Malta
Member Since: October 20 2016
Genres: Biography, History
Books:
id: 251 Title: Steve Jobs
id: 360 Title: Steve Jobs
id: 625
Name: Chad Harbach
Year of Birth: 1986
Place of Birth: Mauritius
Member Since: March 15 2016
Genres: Fiction
Books:
